std/range/package.d

```diff
@@ -317,16 +317,17 @@ if (isBidirectionalRange!(Unqual!Range))
 
 
 ///
-@safe unittest
+pure @safe nothrow @nogc unittest
```

Comment:
Is there already a [D-style](http://dlang.org/dstyle.html) way to order for attributes?
Would probably make sense to do so and thus have a common style in Phobos.